src/components/VizbeeMiniCastBar/hooks/useMiniCastBarActionsHook.js

```diff
@@ -0,0 +1,41 @@
+import { VizbeeManager } from "../../../../";
```

Comment:
Is this a react custom hook? We are not using any react APIs like `useState` or `useEffect` or anything.

Reply:
There was no need of it in this hook